sectionproperties/post/post.py

```diff
@@ -18,18 +19,21 @@ def setup_plot(ax, pause):
         plt.ioff()
 
 
-def finish_plot(ax, pause, title=''):
+def finish_plot(ax, pause, title="", size=1000, dpi=96):
     """Executes code required to finish a matplotlib figure.
 
     :param ax: Axes object on which to plot
     :type ax: :class:`matplotlib.axes.Axes`
     :param bool pause: If set to true, the figure pauses the script until the window is closed. If
         set to false, the script continues immediately after the window is rendered.
-    :param string title: Plot title
+    :param str title: Plot title
```

Comment:
You added `size` and `dpi` into the default arguments. Should they be added to the docstring as well? It's obvious for anyone who's worked with matplotlib, but the same could be said for the `title` arg...

```python
:param int size: Plot size
:param int dpi: Plot dpi
```

Reply:
100% Agree. Thanks for the catch!